versions.yaml

```diff
@@ -281,7 +281,7 @@ languages:
       description: |
         'newest-version' is the latest version known to work when
         building Kata
-      newest-version: "1.54.0"
+      newest-version: "1.56.1"
```

Comment:
OOI, why are we updating rust here? I see the " Update to Rust 2021 edition" commit but it doesn't explain *why*.

We've had some problems changing that version recently, but that was going backwards ;) For refernce though: https://github.com/kata-containers/tests/pull/4192

/cc @fidencio

Reply:
Bumping the newest version is super fine, I would like to not bump the minimum one though, not unless we make a decision on what's the minimum supported one and how we can ensure our code builds with minimum, maximum, and everything in between those.  However, that's more a requirement for the moment this code reaches `main`.  For `ccv0` I'd say we're good to go with the bump.

Reply:
The reason we update rust is due to we use tonic to build GRPC client to talk with attestation agent, and tonic require newer version of rust. I'll add this info to the commit message and change the version to `1.56.0`